web/src/components/config.ts

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+
+export type Config = {
+  PortNaming: {
+    Enable: boolean;
+    PortNames: string[];
+  };
+};
+
+export const ConfigContext = React.createContext(<Config>{ PortNaming: { Enable: true, PortNames: [] } });
```

Comment:
I didn't know about react context, that's interesting!

Reply:
Cool, I will try that way also for topology controller :+1: